packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -327,7 +327,7 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
   /// Triggers a frame after `duration` amount of time.
   ///
   /// This makes the framework act as if the application had janked (missed
-  /// frames) for `duration` amount of time, and then received a v-sync signal
+  /// frames) for `duration` amount of time, and then received a Vsync signal
```

Comment:
nit:

```suggestion
  /// frames) for `duration` amount of time, and then received a vsync signal
```

Reply:
In the documentation, capital "Vsync" is used and I followed that. Should it be lower "vsync"?

https://github.com/flutter/flutter/blob/01953a101576a64aeaec48d434e780ce31e50538/packages/flutter/lib/src/scheduler/binding.dart#L764